applications/FluidDynamicsApplication/python_scripts/apply_inlet_process.py

```diff
@@ -18,16 +18,20 @@ def __init__(self, Model, settings):
             "variable_name"   : "VELOCITY",
             "modulus"         : 0.0,
             "constrained"     : true,
-            "direction"       : "automatic_inwards_normal",
+            "direction"       : [1.0,0.0,0.0],
             "interval"        : [0.0,"End"]
         }
         """)
 
         # Trick: allow "modulus" and "direction" to be a double or a string value (otherwise the ValidateAndAssignDefaults might fail)
-        if(settings.Has("modulus")):
-            if(settings["modulus"].IsString()):
+        if (settings.Has("modulus")):
+            if (settings["modulus"].IsString()):
                 default_settings["modulus"].SetString("0.0")
```

Comment:
How should we proceed then? IMO this can work as a temporary solution until AcceptAsString() and AcceptAsNumber() are implemented.